### ISSUE TITLE
Add optional validation against d.o project listing

### ISSUE THREAD

**Issue by minorOffense** [coldfrontlabs/fserver#18]
To help avoid naming conflicts between feature projects and d.o projects, add a validation check when creating projects against the list of modules on d.o

Make this configuration in the feature server settings.

**Comment by minorOffense** [comment]
@ClassicCut did you implement this already elsewhere?

**Comment by ClassicCut** [comment]
@minorOffense yeah, but not in fserver.  And I also don't really like how I did it...
